lib/dao.ts

```diff
@@ -97,15 +97,15 @@ export class DAO {
     const controller = this.controller;
     const avatar = this.avatar;
     const arcTypesMap = new Map<Address, string>(); // <address: Address, name: string>
-    const contracts = await Contracts.getDeployedContracts();
+    const wrapperService = WrapperService.wrappersByType;
```

Comment:
this actually feels less clear to me. seems like outside people are going to be confused by the term "wrapper". Wrapper of what? dont they want to be interacting with the contracts?

Reply:
The docs describe wrappers right up front as wrapping Arc contracts.

Reply:
An alternative I've been seriously considering:

Rename:

UpgradeScheme =>  UpgradeSchemeFactory
UpgradeSchemeWrapper => UpgradeScheme